cdp-agentkit-core/typescript/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+- Added Farcaster toolkit, tool framework, and example chatbot.
```

Comment:
This change pertains to the `farcaster-langchain` package, let's move it to the CHANGELOG.md there

Reply:
@derekbrown looks like this change stuck around